ManatiCR/manatime
# Manatime-3: Clients content type.

**Step to tests**
- Run the script manatime_local_install.sh 
- See if the features manatime_clients was added and activated.
- Add a new client node

**DoD**
- As an administrator I should be able to go to node/add/client and add a new client with the described fields
- As an administrator, I should be able to view a client (no themed) and click on the Edit tab and update that client.
